src/Core/System/SalesChannel/Event/SalesChannelContextCreatedEvent.php

```diff
@@ -12,21 +12,32 @@
 class SalesChannelContextCreatedEvent extends Event implements ShopwareSalesChannelEvent
 {
     public function __construct(
-        private readonly SalesChannelContext $salesChannelContext,
-        private readonly string $usedToken
+        public readonly SalesChannelContext $context,
```

Comment:
please do not change the mechanics of the event here yet. this should be done as an own PR and for all events and probably along with an ADR.
it would be strange, if some events behave like this and some don't

Reply:
Don't get what you mean. I justed changed the visibility and provided the session data. If you don't want to move it in this direction it would be very polite to just import the PR and revert this change.